### DISCUSSION TITLE
Can't signup using Github

### DISCUSSION THREAD

**Discussion by Etchelon** [vercel/community#1262] [Help] [answered]
Hello,

I'm unable to signup to Vercel via Github. Account not found:

<img width="296" alt="image" src="https://user-images.githubusercontent.com/4141889/209406791-d3e384c7-be4f-4f2d-ac29-72ac542d6b4d.png">

Not sure what to do or who to contact...

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble with signup @Etchelon. Please contact Vercel Support by emailing [registration@vercel.com](mailto:registration@vercel.com) and they'll help you get it sorted out. Be sure include a description of the project you are looking to deploy with Vercel, which will help speed up the account approval process.

**Reply by Etchelon** [reply]
Thanks amy for the swift reply!
Have a merry Xmas 🎄

**Reply by amyegan** [reply]
Merry Christmas to you as well! ☃️
